### PR TITLE
Semester2 us13 draft UI

### PR DESCRIPTION
**Pull request by csenz**
Holds Emma's previous sign in PR and the rest of the updated UI and draft.

### CODE REVIEW

File: src/App.js
Comment (JBorden5) [inline src/App.js]: this will need to be behind an `AuthenticatedRoute` once the auth changes are merged in

File: src/Components/AllPlayerFeed/AllPlayerFeed.js
Comment (JBorden5) [inline src/Components/AllPlayerFeed/AllPlayerFeed.js]: This will need to be updated to use the `useAuthenticatedFetch` hook

**Review by JBorden5 — CHANGES_REQUESTED** [review]
We should wait for the auth changes to get merged in then update this PR to use the new auth components and hooks

File: src/App.js
Comment (JBorden5) [inline src/App.js]: there isn't a _problem_ with adding these comments, but they're also not necessary. I would suggest we remove them, or at least put them above the routes to match comment conventions

File: src/Components/DraftFeed/Components/DraftablePlayers/DraftablePlayers.js
Comment (JBorden5) [inline src/Components/DraftFeed/Components/DraftablePlayers/DraftablePlayers.js:4]: Having a variable outside of a component like this can cause problems for react, including memory leaks. We should move this to the containing component (`DraftFeed` I believe) and then have this component take in an event handler that it uses to update the list.

Let me know if that makes sense. If not, I'd be happy to find time to hop on a video call and do some pair programming on this.

File: src/Components/DraftFeed/Components/DraftablePlayers/DraftablePlayers.js
Comment (JBorden5) [inline src/Components/DraftFeed/Components/DraftablePlayers/DraftablePlayers.js]: This name should probably be `DraftablePlayer` since it represents a single player

File: src/Components/DraftFeed/Components/DraftablePlayers/DraftablePlayers.js
Comment (JBorden5) [inline src/Components/DraftFeed/Components/DraftablePlayers/DraftablePlayers.js:21]: it doesn't look like this is doing anything. I think we can remove it

File: src/Components/DraftFeed/Components/DraftablePlayers/DraftablePlayers.js
Comment (JBorden5) [inline src/Components/DraftFeed/Components/DraftablePlayers/DraftablePlayers.js]: we probably should move the `<button>` outside of the `<h4>` since the plus symbol isn't really part of a title

File: src/Components/DraftFeed/DraftFeed.js
Comment (JBorden5) [inline src/Components/DraftFeed/DraftFeed.js:55]: do we want the 'other teams' to be assigned players at random? from my understanding the user would be drafting elsewhere and just duplicating the picks into our system. We definitely need a way to set other players as 'taken', but I'm not sure it should be random

Comment (csenz) [reply src/Components/DraftFeed/DraftFeed.js:55]: I agree it should not be random, but this was just a placeholder to show off the UI. The UI was the first user story then the logic was another so this will be altered later when we figure out how we are implementing the draft.

File: src/App.js
Comment (csenz) [reply src/App.js]: Resolved

File: src/Components/DraftFeed/Components/DraftablePlayers/DraftablePlayers.js
Comment (csenz) [reply src/Components/DraftFeed/Components/DraftablePlayers/DraftablePlayers.js]: Resolved

File: src/Components/DraftFeed/Components/DraftablePlayers/DraftablePlayers.js
Comment (csenz) [reply src/Components/DraftFeed/Components/DraftablePlayers/DraftablePlayers.js]: Resolved

**Pull request merged**
